### PR TITLE
Adding td element

### PR DESCRIPTION
**Pull request by kingsimmy**
Currently after creating an HtmlTrElement it is not possible to add the children as there is no HtmlTdElement in the library.

Would it be possible to merge this pull request and push a new version into Nuget?

Thanks

### CODE REVIEW

File: src/src/HtmlGenerator/Tag.cs [hunk 1]
```diff
@@ -119,6 +119,7 @@ public static partial class Tag
 		public static HtmlTimeElement Time => new HtmlTimeElement();
 		public static HtmlTitleElement Title => new HtmlTitleElement();
 		public static HtmlTrElement Tr => new HtmlTrElement();
+		public static HtmlTdElement Tr => new HtmlTdElement();
```
Comment (hughbe) [inline src/src/HtmlGenerator/Tag.cs]: This looks wrong - should it be named 'Td' instead of 'Tr'

Thanks

**Comment by kingsimmy** [comment]
Ah yeah, sorry. I've corrected Tag.cs now.

Thanks

**Comment by kingsimmy** [comment]
Hi @hughbe 
Is it okay for you to merge this and update Nuget?

**Comment by hughbe** [comment]
@kingsimmy no problem! Thanks a lot for this - good addition.

I'm sorry but I can't update the nuget package as I'm travelling across Europe (no laptop) until 5th August. I'm sorry if this is a problem - if it is, you can build a local copy of the library and reference it 

Thanks again, and sorry

**Pull request merged**